cmd/syncthing/monitor.go

```diff
@@ -430,3 +441,39 @@ func childEnv() []string {
 	env = append(env, "STMONITORED=yes")
 	return env
 }
+
+// maybeReportPanics tries to figure out if crash reporting is on or off,
+// and reports any panics it can find if it's enabled. We spend at most
+// panicUploadMaxWait uploading panics...
+func maybeReportPanics() {
+	// Try to get a config to see if/where panics should be reported.
+	cfg, err := loadOrDefaultConfig()
```

Comment:
Yes. Why not? :) Typically this will mean something on first startup only, where we will use the same defaults that will be written out when the real process starts.